controller/src/main/java/org/jboss/as/controller/operations/global/GlobalOperationHandlers.java

```diff
@@ -627,17 +628,20 @@ public void handleResult(OperationContext.ResultAction resultAction, OperationCo
         private final ModelNode operation;
         private final ModelNode result;
         private final OperationStepHandler handler; // handler bypassing further wildcard resolution
+        private final boolean checkAlias;
```

Comment:
I merged it anyway, but please delete it later. No one else will be brave enough to touch this hack in order to delete it. ;)